compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -257,6 +257,9 @@ def TileAndDistributeToWorkgroups :
     Option<"maxWorkgroupParallelDims", "max-workgroup-parallel-dims", "int32_t",
       /*default=*/ "",
       "Maximum number of dims to distribute workgroups across.">,
+    Option<"distributionMethod", "distribution-method", "int32_t",
```

Comment:
I think if  you include the right file in `PassDetail.h` you can use the enum type here, but I havent tried it. So this is fine for me.

Reply:
I tried but couldn't pass an integer in the command line in this case.